0.1.15: Incomaptible with Java 8

Build fails with the following error:

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'sg-ff2-parent'.
> Could not resolve all artifacts for configuration ':classpath'.
   > Could not resolve com.github.psxpaul:gradle-execfork-plugin:0.1.15.
     Required by:
         project : > com.github.psxpaul.execfork:com.github.psxpaul.execfork.gradle.plugin:0.1.15
      > No matching variant of com.github.psxpaul:gradle-execfork-plugin:0.1.15 was found. The consumer was configured to find a runtime of a library compatible with Java 8, packaged as a jar, and its dependencies declared externally, as well as attribute 'org.gradle.plugin.api-version' with value '7.0' but:
          - Variant 'apiElements' capability com.github.psxpaul:gradle-execfork-plugin:0.1.15 declares a library, packaged as a jar, and its dependencies declared externally:
              - Incompatible because this component declares an API of a component compatible with Java 11 and the consumer needed a runtime of a component compatible with Java 8
              - Other compatible attribute:
                  - Doesn't say anything about org.gradle.plugin.api-version (required '7.0')
          - Variant 'runtimeElements' capability com.github.psxpaul:gradle-execfork-plugin:0.1.15 declares a runtime of a library, packaged as a jar, and its dependencies declared externally:
              - Incompatible because this component declares a component compatible with Java 11 and the consumer needed a component compatible with Java 8
              - Other compatible attribute:
                  - Doesn't say anything about org.gradle.plugin.api-version (required '7.0')
```

Might be an issue similiar to https://github.com/JetBrains/markdown/issues/70 ?

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -18,6 +18,12 @@ plugins {
 group = "com.github.psxpaul"
 version = File(rootDir, "VERSION").readText().trim()
 
+java {
+    toolchain {
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+}
+
 dependencies {
     implementation(gradleApi())
     implementation("org.jetbrains.kotlin:kotlin-stdlib:1.4.32")

